src/Makefile

```diff
@@ -735,22 +744,31 @@ endif
         clang-profile-use clang-profile-make
 
 build: config-sanity
-	$(MAKE) ARCH=$(ARCH) COMP=$(COMP) all
+	$(MAKE) ARCH=$(ARCH) COMP=$(COMP) \
+	EXTRACXXFLAGS=' $(BLASDEFINE) $(BLASCXXFLAGS) -fopenmp ' \
```

Comment:
I think one can now remove the LEARNCXXFLAGS, LEARNLDFLAGS, EXTRACXXFLAGS and EXTRALDFLAGS and just 
CXXFLAGS +=  $(BLASDEFINE) $(BLASCXXFLAGS) -fopenmp
LDFLAGS += $(BLASLDFLAGS) -fopenmp
after the BLAS section.

Reply:
EXTRACXXFLAGS and EXTRALDFLAGS seems to be required to make profile builds in a reasonable way, otherwise profile builds would overwrite user parameters passed in CXXFLAGS and LDFLAGS. I agree with removal of other FLAGS variables.

Reply:
I don't understand the purpose of DEPENDLDFLAGS

Reply:
DEPENDFLAGS ? Well, we need it for generating the dependencies..but maybe you mean, why it isn't equal to CXXFLAGS? This I don't know.

You're right about EXTRACXXFLAGS and EXTRALDFLAGS